packages/widgets-internal/roi/aprHelper.ts

```diff
@@ -1,8 +1,9 @@
-import { Percent } from "@pancakeswap/sdk";
-import { formatFraction } from "@pancakeswap/utils/formatFractions";
-
-export function getAccrued(principal: number, apr: Percent, timesCompounded = 1, stakeFor = 1) {
-  const aprAsDecimal = parseFloat(formatFraction(apr.asFraction, 6) || "0");
+export function getAccrued(principal: number, apr: number, compoundEvery = 0, stakeFor = 1, compoundOn = false) {
```

Comment:
do we need the `compoundOn` parameter? Does it mean the same if we set `compoundEvery = 0`?

Reply:
yes,let me change it